server.js

```diff
@@ -14,12 +16,19 @@ app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')))
 
 app.use('/files', express.static(`${__dirname}/public`));
 
-app.get('/products/:id', (req, res) => {
+
+app.get('/product/:id', (req, res) => {
   res.sendFile(`${__dirname}/public/index.html`);
 });
 
+app.post('/api/cart/:productid/:userid', (req, res) => {
```

Comment:
You most likely dont need this route since it will be added through the proxy middleware, coming from you components service